cpu/stm32w108/hal/micro/cortexm3/mfg-token.h

```diff
@@ -20,10 +20,13 @@
  * that point to the correct location in the Info Blocks.  (This is the
  * extern, the actual definition is found in hal/micro/cortexm3/token.c)
  *
- * \param name: The name of the token.
- *
- * \param TOKEN_##name##_ADDRESS: The address in EEPROM at which the token
- * will be stored.  This parameter is generated with a macro above.
```

Comment:
This documentation should be fixed rather then removed. There are too many people who don't know the CPP well enaugh. Maybe add a todo.